web/cobrands/northamptonshire/assets.js

```diff
@@ -408,8 +412,12 @@ var northants_defaults = $.extend(true, {}, fixmystreet.assets.alloy_defaults, {
     },
     asset_not_found: function() {
       $("#overlapping_features_msg").addClass('hidden');
-      disable_report_form();
-      show_responsibility_error('#js-not-an-asset', this.fixmystreet.asset_item, this.fixmystreet.asset_type);
+      var emergency_state = ncc_is_emergency_category();
+
+      if (!emergency_state.relevant) {
```

Comment:
As with road not found, should this only be around the error showing line? The form will already be disabled?